### PR TITLE
ad - Using the totalUsers field from CommonsPlus entity to display number of players in a common

### LINKED ISSUE #27: Add a variable for how many players are in a common
Image

NOTE FROM CONRAD:  Do not add a variable.  This variable already exists, and is available through the api point 
`/api/commons/allplus`

But, if you can change the component that is populating this page to one that uses the `/api/commons/allplus` then you can certainly fill in this missing information.

### PR DESCRIPTION
**Pull request by akdas16**
I made the appropriate changes as prof Conrad suggested earlier, and this time I did not introduce a new variable in the Commons entity

Closes #27

### CODE REVIEW

**Comment by akdas16** [comment]
I am stuck at CommonOverview frontend testing. I don't know how to get the tests to work. Any kind of help would be greatly appreciated!!

**Comment by akdas16** [comment]
Screen Shot 2022-11-30 at 3 50 22 PM

Screen Shot 2022-11-30 at 3 50 47 PM

This is the issue I am getting in the CommonsOverview test

File: frontend/src/tests/components/Commons/CommonsOverview.test.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ import AxiosMockAdapter from "axios-mock-adapter";
 
 import CommonsOverview from "main/components/Commons/CommonsOverview"; 
 import PlayPage from "main/pages/PlayPage";
-import commonsFixtures from "fixtures/commonsFixtures"; 
+//import commonsFixtures from "fixtures/commonsFixtures";
```
Comment (pconrad) [inline frontend/src/tests/components/Commons/CommonsOverview.test.js:9]: Let's remove commented out code.

File: frontend/src/tests/components/Commons/CommonsOverview.test.js [hunk 2]
```diff
@@ -59,13 +63,19 @@ describe("CommonsOverview tests", () => {
     });
 
     test("No LeaderboardPage for an ordinary user when commons has showLeaderboard = false", async () => {
-        const ourCommons = {
+        /*const ourCommons = {
             ...commonsFixtures.oneCommons,
             showLeaderboard : false
+        };*/
+        const ourCommons = {
```
Comment (pconrad) [inline frontend/src/tests/components/Commons/CommonsOverview.test.js:70]: If this variable isn't used, we should delete it...

**Pull request closed without merging**